### PR TITLE
Quantization docs

### PR DESCRIPTION
**Pull request by dame-cell**

### CODE REVIEW

File: docs/source/conceptual_guides/quantization.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+* The `AdaLora` techniques also allows us to [quantize and fine-tune models](https://huggingface.co/docs/peft/main/en/package_reference/adalora) but unlike LoRA, which distributes parameters evenly across all modules. More parameters are budgeted for important weight matrices and layers while less important ones receive fewer parameters.
+
+* The `ia3` allows us to [quantize and fine-tune models](https://huggingface.co/docs/peft/main/en/package_reference/ia3) it is a method that adds three learned vectors to rescale the keys and values of the self-attention and encoder-decoder attention layers, and the intermediate activation of the position-wise feed-forward network.
+
+
+## References
+
+- [All about Quantization ](https://www.youtube.com/watch?v=UQlsqdwCQdc&t=478s) you-tube video
+- [LLM Series — Parameter Efficient Fine Tuning](https://medium.com/@abonia/llm-series-parameter-efficient-fine-tuning-e9839fae44ac)
```
Comment (younesbelkada) [inline docs/source/conceptual_guides/quantization.md:60]:
```suggestion
- [LLM Series — Parameter Efficient Fine Tuning](https://medium.com/@abonia/llm-series-parameter-efficient-fine-tuning-e9839fae44ac)
- [🤗 transformers quantization documentation page](https://huggingface.co/docs/transformers/main_classes/quantization) 
```

**Review by younesbelkada — COMMENTED** [review]
Looks great thanks ! I left one suggestion, can you also fix the merge conflict?

File: docs/source/conceptual_guides/quantization.md (hunk 1 above)
Comment (dame-cell) [reply docs/source/conceptual_guides/quantization.md:60]: I'll do that as soon as possible

Comment (dame-cell) [reply docs/source/conceptual_guides/quantization.md:60]: I'll do that as soon as possible

Comment (younesbelkada) [reply docs/source/conceptual_guides/quantization.md:60]: you can just click into `"Commit Suggestion"` and the suggestion will be added directly

**Comment by dame-cell** [comment]
Hey I did it 
Hopefully it works now

File: docs/source/_toctree.yml [hunk 2]
```diff
@@ -58,6 +58,17 @@
...
+    title : Quantization
+
+- title: Reference
+  sections:
+  - local: package_reference/peft_model
+    title: PEFT model
+  - local: package_reference/config
+    title: Configuration
+  - local: package_reference/tuners
+    title: Tuners
```
Comment (younesbelkada) [inline docs/source/_toctree.yml:71]:
```suggestion
```

**Review by younesbelkada — COMMENTED** [review]
Thank you! I think the sections below are not needed ! Otherwise looks great IMO

**Pull request closed without merging**